app/src/main/res/values/styles.xml

```diff
@@ -1,13 +1,13 @@
 <resources>
 
-    <style name="Hedvig.Theme" parent="Hedvig.Theme.V21" />
+    <style name="Hedvig.Theme" parent="Hedvig.Theme.V23" />
 
     <style name="Hedvig.Theme.KeyGear">
         <item name="android:navigationBarColor">@color/translucent_background</item>
     </style>
 
     <style name="Hedvig.Theme.LoggedIn">
-        <item name="android:navigationBarColor">@color/color_background_95</item>
+        <item name="android:navigationBarColor">#01FF0000</item>
```

Comment:
What's this color?

Reply:
This is the almost transparent hack

Reply:
Is it required on this lower API level? Does the OS enforce contrast without it?

If yes, we should in the very least make sure it's not a very transparent red but probably a very transparent black

Reply:
Okey, this was apparently not needed when tested on api 23, so it is now changed to `@android:color/transparent`